style.css

```diff
@@ -43,6 +43,10 @@ header {
   width: 16px;
 }
 
+#nav-links {
```

Comment:
### **I like your CSS skills. GOOD JOB 🎉🎊🙌. Keep on building projects like this one 🚀**

Reply:
@Rustamxon7 thank you so much, your comment boosted my confidence 😄